hw/mcu/dialog/da1469x/src/da1469x_lpclk.c

```diff
@@ -38,6 +38,7 @@ static void
 da1469x_lpclk_settle_tmr_cb(void *arg)
 {
     da1469x_clock_lp_xtal32k_switch();
+    da1469x_clock_lp_xtal32k_calibrate();
```

Comment:
Calibration of XTAL32K should be optional and disabled by default - we should assume after settling it's accurate and has its native frequency.

Reply:
See below...

Reply:
Done.